packages/components/stepper/src/js/step-form/step-form.controller.js

```diff
@@ -62,6 +62,7 @@ export default class StepFormController {
   }
 
   onFormSubmit(form) {
+    this.$scope.$emit('oui-step-form.submit', { form });
```

Comment:
You can use a callback function instead, this way, the day we get rid of AngularJS, the code still works :)

Reply:
I'm re-applying #402 changes;)

Reply:
@FredericEspiau FYI events can be reproduced in other technologies (such as Vue), if you have a better solution in mind for listening to all those submit changes I'm all ears